src/dmd/backend/elfobj.d

```diff
@@ -3378,10 +3392,26 @@ private void obj_rtinit()
     // make the symbols hidden so that each DSO gets its own brackets
     IDXSYM minfo_beg, minfo_end, dso_rec;
 
+    version (OpenBSD)
```

Comment:
is there any problem in unconditionally emitting `deh_beg` and `deh_end` (and thus remove all the `version (OpenBSD)`?

Reply:
I'm not actually sure about this. This is the place I was hoping to rely on the expertise of the community. My original version of the diff did unconditionally do this. But as I don't have other platforms to test with, I erred on the side of being overly cautious to not alter any other platform.

Reply:
Use `static if (TARGET_OPENBSD)`